packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -58,9 +62,32 @@ class RecoveryPhoneHandler {
 
     await this.customs.check(request, email, 'recoveryPhoneSendCode');
 
-    let status = false;
+    const ftlIdMsgs = [
+      {
+        id: 'recovery-phone-signin-sms-body-part-1',
+        message: 'Mozilla recovery code:',
+      },
+      {
+        id: 'recovery-phone-sms-body-part-2',
+        message: 'Expires in 5 minutes. Do not share this code.',
+      },
+    ];
+    const localizedStrings = await this.localizer.localizeStrings(
+      request.app.locale,
+      ftlIdMsgs
+    );
+
+    const localizedMessageBody = {
+      part1: localizedStrings['recovery-phone-signin-sms-body-part-1'],
+      part2: localizedStrings['recovery-phone-sms-body-part-2'],
+    };
+
+    let success = false;
     try {
-      status = await this.recoveryPhoneService.sendCode(uid);
+      success = await this.recoveryPhoneService.sendCode(
+        uid,
+        localizedMessageBody
```

Comment:
With a template function this would be

(code:string) => `${part1} ${code} ${part2}`